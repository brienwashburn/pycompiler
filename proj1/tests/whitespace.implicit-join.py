month_names = ['J', 'F', 'M',
               'A', 'M', 'J',
               'J', 'A', 'S',
               'O', 'N', 'D']
