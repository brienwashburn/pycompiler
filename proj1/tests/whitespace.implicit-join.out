(ID "month_names")
(PUNCT "=")
(PUNCT "[")
(LIT "J")
(PUNCT ",")
(LIT "F")
(PUNCT ",")
(LIT "M")
(PUNCT ",")
(LIT "A")
(PUNCT ",")
(LIT "M")
(PUNCT ",")
(LIT "J")
(PUNCT ",")
(LIT "J")
(PUNCT ",")
(LIT "A")
(PUNCT ",")
(LIT "S")
(PUNCT ",")
(LIT "O")
(PUNCT ",")
(LIT "N")
(PUNCT ",")
(LIT "D")
(PUNCT "]")
(NEWLINE)
(ENDMARKER)
